saleor/core/utils/random_data.py

```diff
@@ -210,10 +210,14 @@ def create_products_by_class(product_class, schema,
             create_product_images(
                 product, random.randrange(1, 5), class_placeholders)
         variant_combinations = get_variant_combinations(product)
+        prices = get_price_override(schema, len(variant_combinations))
         for i, attr_combination in enumerate(variant_combinations, 1337):
```

Comment:
Why `1337`? Maybe it is too much but in my opinion digits and bools should be passes named:
```python
enumerate(variant_combinations, start=1337)
```

It is way more readable.